README.md

```diff
@@ -1,24 +1,166 @@
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* How to run the test suite
+<!-- PROJECT DESCRIPTION -->
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📖 Project Name
 
-* Deployment instructions
+## PostPulse
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+- `Ruby on Rails`
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- Show the list of posts. 
+- Empower readers to interact with them by adding comments and liking posts.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started
+To clone the repository in local environment try following steps.
+
+### Prerequisites
+
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Node.js and npm for package installation.
+```
+$ ruby --version && gem --version
+```
+### Setup
+
+Clone the repository using [this link](https://github.com/ShinhyoBelok/PostPulse.git).
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/ShinhyoBelok/PostPulse.git
+```
+### Get into development
+
+In the terminal, run these commands to get into development.
+```
+$ cd catalogue-using-ruby
+
+$ bundle init
+
+$ bundle install (install dependencies)
+
+$ ruby main.rb (run the app locally)
+```
+### Usage
+
+This website is applicable for both mobile and desktop version.
+
+### Deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Shinhyo Belliard**
+
+- GitHub: [@ShinhyoBelok](https://github.com/ShinhyoBelok)
+- Twitter: [@Bell_iraki](https://twitter.com/Bell_iraki)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shinhyo-belliard-okazaki-807a38249/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Validations and Model specs.**
+- [ ] **Processing data in models.**
+- [ ] **Setup and controllers.**
+- [ ] **Controllers specs.**
+- [ ] **Views.**
+- [ ] **Forms.**
+- [ ] **Integration specs for Views and fixing n+1 problems.**
+- [ ] **Add Devise.**
+- [ ] **Add authorization rules.**
+- [ ] **Add API endpoints.**
+- [ ] **documentation.**
```

Comment:
- [ ] I notice too many future futures keys. Please Make sure to include 1-3 potential features to be added 😄